scancode

```diff
@@ -105,10 +105,33 @@ _canonicalize_file_path() {
 SCANCODE_BIN="$( realpath "${BASH_SOURCE[0]}" )"
 SCANCODE_ROOT_DIR="$( cd "$( dirname "${SCANCODE_BIN}" )" && pwd )"
 
+SCANCODE_CONFIG_DIR="etc/conf"
+
+GIT_STATUS=$(git -C "$SCANCODE_ROOT_DIR" status --porcelain 2> /dev/null)
```

Comment:
Does this also implicitly checks for Git being available locally? Just curious what the different with a plain `git status`?

Reply:
Exactly. It implicitly checks for `$SCANCODE_ROOT_DIR` to be a Git working tree (`git status` returns code 128 otherwise), and also for the `git` executable to be present (in this case it's Bash that returns code 127).